src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -599,189 +605,189 @@ internal DataTable BuildSchemaTable()
                     Debug.Assert(col.baseTI.metaType != null, @"col.baseTI.metaType should not be null.");
-                    schemaRow[IsLong] = col.baseTI.metaType.IsLong;
+                    schemaRow[isLong] = col.baseTI.metaType.IsLong;
                 }
                 else
                 {
-                    schemaRow[IsLong] = col.metaType.IsLong;
+                    schemaRow[isLong] = col.metaType.IsLong;
                 }
 
                 // mark unique for timestamp columns
                 if (SqlDbType.Timestamp == col.type)
                 {
-                    schemaRow[IsUnique] = true;
-                    schemaRow[IsRowVersion] = true;
+                    schemaRow[isUnique] = true;
+                    schemaRow[isRowVersion] = true;
                 }
                 else
                 {
-                    schemaRow[IsUnique] = false;
-                    schemaRow[IsRowVersion] = false;
+                    schemaRow[isUnique] = false;
+                    schemaRow[isRowVersion] = false;
                 }
 
-                schemaRow[IsReadOnly] = col.IsReadOnly;
-                schemaRow[IsColumnSet] = col.IsColumnSet;
+                schemaRow[isReadOnly] = col.IsReadOnly;
+                schemaRow[isColumnSet] = col.IsColumnSet;
 
-                if (!ADP.IsEmpty(col.serverName))
+                if (!string.IsNullOrEmpty(col.serverName))
                 {
-                    schemaRow[BaseServerName] = col.serverName;
+                    schemaRow[baseServerName] = col.serverName;
                 }
-                if (!ADP.IsEmpty(col.catalogName))
+                if (!string.IsNullOrEmpty(col.catalogName))
                 {
-                    schemaRow[BaseCatalogName] = col.catalogName;
+                    schemaRow[baseCatalogName] = col.catalogName;
                 }
-                if (!ADP.IsEmpty(col.schemaName))
+                if (!string.IsNullOrEmpty(col.schemaName))
                 {
-                    schemaRow[BaseSchemaName] = col.schemaName;
+                    schemaRow[baseSchemaName] = col.schemaName;
                 }
-                if (!ADP.IsEmpty(col.tableName))
+                if (!string.IsNullOrEmpty(col.tableName))
                 {
-                    schemaRow[BaseTableName] = col.tableName;
+                    schemaRow[baseTableName] = col.tableName;
                 }
-                if (!ADP.IsEmpty(col.baseColumn))
+                if (!string.IsNullOrEmpty(col.baseColumn))
                 {
-                    schemaRow[BaseColumnName] = col.baseColumn;
+                    schemaRow[baseColumnName] = col.baseColumn;
                 }
-                else if (!ADP.IsEmpty(col.column))
+                else if (!string.IsNullOrEmpty(col.column))
                 {
-                    schemaRow[BaseColumnName] = col.column;
+                    schemaRow[baseColumnName] = col.column;
```

Comment:
NB: Thank you for these changes. I'd be losing my mind if I had to work on this code trying to figure out each time if it was referencing a property or a variable.